integration-tests/tests/src/node/index.ts

```diff
@@ -16,7 +16,14 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
-// NOTE: This file is only supposed to be imported from a Node.js environment
+if (typeof process.versions.node !== "string") {
+  throw new Error("This file is only supposed to be imported from a Node.js environment!");
```

Comment:
Asserting that this gets imported from node instead of simply commenting the intent.